.github/workflows/unit_tests.yml

```diff
@@ -29,4 +29,4 @@ jobs:
     - name: Run unit tests
       run: |
         set -euo pipefail
-        pytest .
\ No newline at end of file
+        ./venv/bin/pytest .
```

Comment:
Oh, you need to re-enable the environment, as each `run` is in it's own shell. See https://github.com/google/model-transparency/blob/e1d31871e88b6e8c2b821fd97786c92f9a4a03f8/.github/workflows/slsa_for_ml.yml#L43

Reply:
what a mistake :)